modules/docker/docker_compose.go

```diff
@@ -46,6 +50,15 @@ func runDockerComposeE(t testing.TestingT, stdout bool, options *Options, args .
 
 	result := icmd.RunCmd(dockerComposeVersionCmd)
 
+	if options.EnableBuildKit {
+		if options.EnvVars == nil {
+			options.EnvVars = make(map[string]string)
+		}
+
+		options.EnvVars["DOCKER_BUILDKIT"] = "1"
+		options.EnvVars["COMPOSE_DOCKER_CLI_BUILD"] = "1"
```

Comment:
Why is this COMPOSE_DOCKER_CLI_BUILD instead of DOCKER_COMPOSE_CLI_BUILD ?

Reply:
Ah, I understand:

https://www.docker.com/blog/faster-builds-in-compose-thanks-to-buildkit-support/

> As docker-compose passes its environment variables to the Docker CLI, we can also tell the CLI to use BuildKit instead of the default builder. To accomplish that, we can execute this:

```
$ COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker-compose build
```